packages/react-scripts/fixtures/kitchensink/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src",
+    "paths": {
+      "@": ["src"]
```

Comment:
Perhaps add a test with `*` along with `@*` for compatibility with the previous `NODE_PATH` approach.

Reply:
For people that had their `NODE_PATH` set to `.` and were importing like this: `src/components/button`?

Reply:
So, I think it should be the below as a default, as that matches the `NODE_PATH` behaviour. Assuming the user set `NODE_PATH` to `src` would make this a good default:
```
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "*": ["src/*"],
    }
  },
```
Then, a user could do the following, if they chose.
```
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@*": ["src/*"],
    }
  },
```

Reply:
Hmm, I'm not sure if we should provide a default as the `NODE_PATH` is not set to `.` by default which would potentially confuse users thinking that's default node behavior.

Reply:
I understand, by default there's no NODE_PATH. However, I think most people that use it would use it with `src`.

As per most implementations, you can leave `"baseUrl"` as `"."`, and the aliasing itself can happen in paths.